services/src/Microsoft.Azure.IIoT.Services.Processor.Events/src/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,8 @@
+// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Azure.IIoT.OpcUa.Graph.Tests")]
```

Comment:
Wrong namespace for this assembly.   Probably copy and paste, --- just remove the entire AssemblyInfo.cs from all processors as there are no unit tests.